.github/workflows/test.main.kts

```diff
@@ -117,17 +131,62 @@ private fun validateTypings() {
         { it.owner == "DamianReeves" && it.name == "write-file-action" },
     )
 
+    println()
+    val actions = baseRef.let { baseRef ->
```

Comment:
Since this piece grew significantly, let's put it in a function like `listActionsToValidate`. Inside, we can further split to `listAllActionsInRepo` and `listModifiedActions`.

Reply:
Done.